### PR TITLE
Feat/error metrics

### PR DESCRIPTION
**Pull request by CiottiGiorgio**

### CODE REVIEW

**Comment by CiottiGiorgio** [comment]
Bumping this because it risks getting stale. I think the only blocker for this PR is that we needed to confirm that `lastUpdate` can be successfully used in a sensible way in the metrics UI. If that's the case, can we merge?

**Comment by solanoepalacio** [comment]
> Bumping this because it risks getting stale. I think the only blocker for this PR is that we needed to confirm that `lastUpdate` can be successfully used in a sensible way in the metrics UI. If that's the case, can we merge?

yup, that's exactly the case. I just want to verify that before we merge to avoid having to redo a part of this later on. I wanted to take care of this yesterday, but I didn't find the time :( 

Can you take some time this morning and try to clear this out?

File: src/multi-wallet-exporter.ts [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+  createExporterErrorCounter, createExporterLastUpdate
+} from './wallet-exporter';
 
 export type MultiWalletExporterOptions = MultiWalletWatcherOptions & {
   prometheus?: PrometheusOptions
 }
 export class MultiWalletExporter extends MultiWalletWatcher {
   private gauge: Gauge;
+  private errorCounters: Map<string, Counter> = new Map();
+  private lastUpdate: Gauge;
```
Comment (solanoepalacio) [inline src/multi-wallet-exporter.ts:19]: how about `refreshTimestamp` (or `refreshDate`)? `lastSuccesfullPoll`? `balanceRefreshTimestamp`?

Comment (solanoepalacio) [reply src/multi-wallet-exporter.ts:19]: `lastRefreshTimestamp`?

**Review by solanoepalacio — APPROVED** [review]

**Comment by CiottiGiorgio** [comment]
Should we bump the package version for this PR?

**Pull request merged**